### PR TITLE
Update the spec test and turn on SIMD.

### PR DESCRIPTION
**Pull request by q82419**

### CODE REVIEW

File: tools/wasmedge/wasmedger.cpp [hunk 1]
```diff
@@ -125,11 +125,11 @@ int main(int Argc, const char *Argv[]) {
     Conf.removeProposal(WasmEdge::Proposal::ReferenceTypes);
   }
   if (PropSIMD.value()) {
-    Conf.addProposal(WasmEdge::Proposal::SIMD);
-  }
-  if (PropAll.value()) {
-    Conf.addProposal(WasmEdge::Proposal::SIMD);
+    Conf.removeProposal(WasmEdge::Proposal::SIMD);
   }
+  /// Left for the future proposals.
```
Comment (hydai) [inline tools/wasmedge/wasmedger.cpp:130]: Why do you comment these out?

**Review by hydai — CHANGES_REQUESTED** [review]

File: tools/wasmedge/wasmedger.cpp (hunk 1 above)
Comment (q82419) [reply tools/wasmedge/wasmedger.cpp:130]: We've turned on the all proposals we implemented now.

Comment (hydai) [reply tools/wasmedge/wasmedger.cpp:130]: Why not delete them directly?

Comment (q82419) [reply tools/wasmedge/wasmedger.cpp:130]: I think we should keep this for the proposals we will implement and not merge into the official spec in the future?

**Review by hydai — APPROVED** [review]

**Pull request merged**